### PR TITLE
REST API v2 - list of images - mandatory Created attribute

### PR DESCRIPTION
**Pull request by psakar**
fixes https://github.com/containers/podman/issues/8418

### CODE REVIEW

**Comment by rhatdan** [comment]
@psakar Thanks for the PR.

You need to sign your PRs

git commit -a --amend -s
git push --force

We also would prefer some more explanation in the check in message.

There is some debate going on whether the clients should handle omitted entries from the json file messages.  But this LGTM
@jwhonce @baude @mheon PTAL

**Comment by psakar** [comment]
signed. Added explanation to commit message

**Comment by rhatdan** [comment]
Lint, does not like your change, I think you just remove the json flag altogether.

File: pkg/domain/entities/images.go [hunk 1]
```diff
@@ -54,7 +54,7 @@ type ImageSummary struct {
 	ID          string            `json:"Id"`
 	ParentId    string            `json:",omitempty"` // nolint
 	RepoTags    []string          `json:",omitempty"`
-	Created     int64             `json:",omitempty"`
+	Created     int64             `json:`
```
Comment (mheon) [inline pkg/domain/entities/images.go:57]: Remove the `json:` tag entirely please

Comment (psakar) [reply pkg/domain/entities/images.go]: done in another commit. Please let me know in case I should squash it

**Comment by rhatdan** [comment]
@psakar Just to add a little bit to your pain, you need to gofmt your code to pass the lint test.

gofmt -w  -s
Most of us have this built into our editors.

**Comment by rhatdan** [comment]
Yes you need to squash.

**Comment by psakar** [comment]
squashed and formatted (though it is a bit surprising for me that lines before change are reformatted)

**Comment by mheon** [comment]
LGTM

**Comment by TomSweeneyRedHat** [comment]
LGTM
@psakar I concur, I don't understand that change from gofmt either.

**Comment by psakar** [comment]
related to https://github.com/containers/podman/issues/7927

**Pull request merged**